runtime/doc/dev_style.txt

```diff
@@ -928,37 +867,10 @@ execute, simply `assert`: >
         assert(false);
     }
 
-Pointer Expressions ~
-
-No spaces around period or arrow. Pointer operators do not have trailing
-spaces.
-
-The following are examples of correctly-formatted pointer and reference
-expressions: >
-
-    x = *p;
-    p = &x;
-    x = r.y;
-    x = r->y;
-
-Note that:
-
-    - There are no spaces around the period or arrow when accessing a member.
-    - Pointer operators have no space after the * or &.
-
 Boolean Expressions ~
 
-When you have a boolean expression that is longer than the standard line
-length, keep operators at the start of the line. >
-
-    if (this_one_thing > this_other_thing
-        && a_third_thing == a_fourth_thing
-        && yet_another && last_one) {
-      ...
-    }
-
-Also note that you should always use the punctuation operators, such as `&&`
-and `~`, rather than the word operators, such as `and` and `compl`.
+You should always use the punctuation operators, such as `&&` and `~`, rather
+than the word operators, such as `and` and `compl`.
```

Comment:
does the latter even exist in c?

Reply:
No idea. I'll just change it to `or` instead.

Reply:
I mean the latter as in any word operator. `and` and `or` does not exist as well (just checked :)

Reply:
Doh. I'll just remove the entire thing then. Thanks.